db/v-3/tables/vulnerabilities/comments.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+
+    <changeSet id="comments-1" author="nulls">
+        <createTable tableName="comments">
```

Comment:
This needs to be removed from the Vulnerabilities directory. Comments are used in other places

Reply:
I guess `tag.xml` should be moved too which potentially can be applied to another entity

Reply:
done